components/Tabs/Tabs.js

```diff
@@ -17,46 +20,29 @@ class TabLink {
 
   };
 
-  select() {
-    // Get all of the elements with the tabs-link class
-    // const links;
-
-    // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
-
-    // Add a class named "tabs-link-selected" to this link
-    // this.element;
-    
-    // Call the select method on the item associated with this link
-
+  onLinkSelect() {
+    this.tabItem.select()
   }
 }
 
 class TabItem {
-  constructor(element) {
+  constructor(item) {
     // Assign this.element to the passed in element
-    // this.element;
+    this.tabItem = item
+    this.allLinkItem = document.querySelectorAll('tabs-item-description')
   }
 
   select() {
-    // Select all ".tabs-item" elements from the DOM
-    // const items;
 
-    // Remove the class "tabs-item-selected" from each element
-    
-    // Add a class named "tabs-item-selected" to this element
-    //this.element;
+    allLinkItem.forEach(function(content){
```

Comment:
Should be `this.allLinkItem` to refer to what you defined in the `TabItem` constructor (line 32).